vignettes/GGIR.Rmd

```diff
@@ -2006,6 +2006,35 @@ can access [this](https://github.com/Maylor8/RadarPlotGenerator) github reposito
 which provides the R code and detailed instructions on how to make the radar 
 plots using your own data.
 
+
+## Minimum recording duration
+
+GGIR has been designed to process multi-day recordings. The minimum recording duration
+considered by GGIR depends on the type of analysis:
+
+**Running part 1 and 2**
+
+- File size; At least 2MB, where 2MB can be adjusted with argument minimumFileSize. 
+This should not be changed unless you have good reason to believe that a smaller 
+file size is also acceptable.
+
+- Recording duration: At least two long epoch windows 
+(default 60 minutes) in g.readaccfile. The size of this epoch can be altered with
+the second and third value of vector argument `windowsizes`, where the third should not
+be smaller than the second. For example, in short lasting lab-experiments you may
+find it easier to set this to `windowsizes = c(5, 600, 600)` as non-wear detection is
+usually not necessary in lab studies.
+
+**Running part 3 and 4**
+
+- At least one night of data is expected, where a night is expected to have at least the timestamp 
+for midnight. If midnight is not found the sleep detection is skipped.
+
```

Comment:
For part 3 and 4, if no midnight is available but recording starts before 4am, then the wakeup time would be calculated and that night analyzed. For example, a polysomnography study in which participant slept from 1am to 8am, I think it would be possible to get that night if setting includenightcrit to low number.

Should this be rephrased like this? 
- At least one night of data is expected, where a night is expected to include some data from midnight to 4 AM. If data from midnight to 4 AM is not found (e.g., recording from 6 AM to 11 PM), the sleep detection is skipped.

Reply:
(we corresponded via email about this, but just to complete my reply to this PR) 

Although we may facilitate exceptions to the midnight requirement for specific study designs or for part 5, I think it is best to keep the communication about part 4 focussed on a sleep research perspective where having a midnight in the data is important. If part 4 outputs wakeup or onset times when midnight is not present in the data then we need to mask this from the part 4 csv reports.